robots:
  a_bot:
    default_gripper: a_bot_gripper
  a_bot:
    default_gripper: b_bot_single_suction_gripper

grippers:
  a_bot_gripper:
    type: PrecisionGripper
    args:
      prefix: a_bot_
  b_bot_single_suction_gripper:
    type: SuctionGripper
    args:
      prefix: b_bot_gripper_
      eject: false

cameras:
  a_phoxi_m_camera: PhoXiCamera
  a_bot_camera: DepthCamera

ftsensor:
  a_bot:
    thresholds:
      force:
        min: [ -5, -5, -5 ]
        max: [ 10, 10, 10 ]
  b_bot:
    thresholds:
      force:
        min: [ -5, -5, -10 ]
        max: [ 10, 10, 10 ]
